docs/install/knative-with-operators.md

```diff
@@ -25,10 +27,55 @@ Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set
 
 ## Limitations of Knative Operators:
 
-Knative Operators use custom resources (CRs) to configure your Knative deployment.
+Knative Operators use custom resources (CRs) to configure your Knative deployment. They have not been tested in a production
+environment, and should be used for development or test purposes only.
```

Comment:
This should probably say that they are currently alpha; we should also start to think about how we remove this warning about non-production use.